common/src/main/java/net/draycia/carbon/common/config/PrimaryConfig.java

```diff
@@ -93,6 +93,12 @@ public class PrimaryConfig {
     @Comment("Whether Carbon's nickname management should be used. Disable this if you wish to have another plugin manage nicknames.")
     private boolean useCarbonNicknames = true;
 
+    @Comment("Minimum number of characters in nickname.")
+    private int nicknameMinLimit = 3;
+
+    @Comment("Maximum number of characters in nickname.")
+    private int nicknameMaxLimit = 16;
```

Comment:
The reason for the default setting of 16 characters is that MinecraftID can be set to a maximum of 16 characters.